pkg/client/cli/cliutil/systema.go

```diff
@@ -93,3 +96,51 @@ func GetCloudAPIKey(ctx context.Context, description string, autoLogin bool) (st
 	}
 	return keyData.GetApiKey(), nil
 }
+
+// GetCloudLicense communicates with system a to get the jwt version of the
+// license, puts it in a kubernetes secret, and then writes that secret to the
+// output file for the user to apply to their cluster
+func GetCloudLicense(ctx context.Context, outputFile, id string) (string, error) {
```

Comment:
The "writing to a file" part seems like something that should be part of the `cli/license.go` subcommand implementation, not something in `cliutil`; it's a mixing of concerns.